core-aam/core-aam.html

```diff
@@ -1528,7 +1528,7 @@
                   <p>If the referenced objects are in the accessibility tree, expose pointers to them using <code>IA2_RELATION_DESCRIBED_BY</code>, and expose reverse relations as described in <a href="#mapping_additional_relations">Relations</a>.</p></td>
                   <td><p>Use in calculating the accessible Description as described in <a href="#mapping_additional_nd">Name Computation</a>.</p>
                   <p>If the referenced objects are in the accessibility tree, expose pointers to the referenced accessible objects using the <code>DescribedBy</code> property.</p></td>
-                  <td><p>Use in calculating the accessible Description as described in <a href="#mapping_additional_nd">Name Computation</a>. Expose in the description property of the accessible object.</p>
+                  <td><p>Use in calculating the accessible Description as described in <a href="#mapping_additional_nd">Name Computation</a>. Expose in the <code>FullDescription</code> property. Expose in the description property of the accessible object.</p>
```

Comment:
Bad location:  the text about FullDescription has been added to the ATK/AT-SPI mapping, not the UIA mapping.